docs/resources/azure_express_route_circuits_resources.md

```diff
@@ -0,0 +1,126 @@
+| service_provider_properties_bandwidth_in_mbps            | A list of The bandwidth in Mbps of the circuit being created on the Service Provider.                                                        | `id`              |
+| service_provider_properties_peering_locations           | A list of The name of the peering location and not the Azure resource location. Changing this forces a new resource to be created.                               | `tag`             |
+| service_provider_properties_names             |   The name of the ExpressRoute Service Provider. Changing this forces a new resource to be created.   | `type` |
+| service_provider_provisioning_states           | The ExpressRoute circuit provisioning state from your chosen service provider. Possible values are "NotProvisioned", "Provisioning", "Provisioned", and "Deprovisioning".                                               | `name`            |
+| service_keys            | State of express circuitHostName creation                                                              | `id`              |
+| stags           | The identifier of the circuit traffic. Outer tag for QinQ encapsulation.                         `Number` | `stag`             |
+| global_reach_enabled             | A list of The ExpressRoute circuit allowGlobalReachEnable   `boolean`                                   `boolean`| `global_reach_enabled`         |
+| allow_global_reach             |   A list of The ExpressRoute circuit Flag denoting global reach status. `boolean`| `allow_global_reach` |
+| gateway_manager_etags           | A list of The GatewayManager Etag.                                              | `tags`            |
+| allow_classic_operations            | A list of Allow classic operations. `boolean`                                                     | `allow_classic_operation`            |
+| circuit_provisioning_states           | A list of State of express circuitHostName creation              `Enabled` or `Disabled`                |     `circuit_provisioning_state`         |
+| sku_names             | A List of Name sku block for the ExpressRoute circuit as documented below.                                      | `sku_name`         |
+| sku_tiers             | A List of  The service tier. Possible values are Basic, Local, Standard or Premium.      | `sku_tier` |
+| sku_family             | A List of  The billing mode for bandwidth. Possible values are MeteredData or UnlimitedData. | `sku_family` |
+<superscript>*</superscript> For information on how to use filter criteria on plural resources refer to [FilterTable usage](https://github.com/inspec/inspec/blob/master/dev-docs/filtertable-usage.md).
+
+
+## Examples
+
+### Ensure that the express_route_circuits resource has is from same type
+```ruby
+describe azure_express_route_circuits_resource(resource_group: 'MyResourceGroup', name: 'bastion_name') do
```

Comment:
these should be examples of plural resource

Reply:
actually all the example name in this file is of singular .. it should be plural

Reply:
Yes Changed all of them.